graph4nlp/pytorch/inference_wrapper/base.py

```diff
@@ -142,15 +144,22 @@ def preprocess(self, raw_contents: list):
                 )
             elif self.data_item_class == Text2LabelDataItem:
                 data_item = self.data_item_class(input_text=raw_sentence, tokenizer=self.tokenizer)
+            # default the data item only need output_text
             else:
+                # data_item = self.data_item_class(
+                #     input_text=raw_sentence, tokenizer=self.tokenizer, output_text=None
+                # )
                 data_item = self.data_item_class(
-                    input_text=raw_sentence, tokenizer=self.tokenizer, output_text=None
+                    input_text=raw_sentence,
+                    output_text=None,
+                    output_tree=None,
+                    tokenizer=self.tokenizer,
                 )
             data_item_collect.append(data_item)
 
         data_item_collect = self.dataset.build_topology(data_items=data_item_collect)
         for i in range(len(data_item_collect)):
             data_item_collect[i] = self.dataset._vectorize_one_dataitem(
-                data_item_collect[i], self.vocab_model, use_ie=use_ie
+                data_item_collect[i], self.vocab_model, use_ie=use_ie, edge_vocab=self.edge_vocab
```

Comment:
But the api in dataset._vectorize_one_dataitem doesn't have the parameter: "edge_vocab", please check it.
[_vectorize_one_dataitem ](https://github.com/graph4ai/graph4nlp/blob/dee086d3f774988b6789dc35a59231bfae599dfc/graph4nlp/pytorch/data/dataset.py#L916)

Reply:
1. move the edge vocab to the library code
2. unify edge_vocab with vocab_model